CHANGELOG.md

```diff
@@ -1,3 +1,89 @@
+# 0.0.119 - XXX - YYY
+
+## API Updates
+ * The LDK crate ecosystem MSRV has been increased to 1.63 (#2681).
+ * The `bitcoin` dependency has been updated to version 0.30 (#2740).
+ * `lightning-invoice::payment::*` have been replaced with parameter generation
+   via `payment_parameters_from[_zero_amount]_invoice` (#2727).
+ * `{CoinSelection,Wallet}Source::sign_tx` are now `sign_psbt` (#2775).
```

Comment:
Maybe should note this is to solve signing with taproot inputs

Reply:
Because for Taproot we need the previous utxo for all the inputs in the tx. The previous api didn't provide that but the psbt does